portafly/src/pages/Overview.tsx

```diff
@@ -1,32 +1,90 @@
-import React from 'react'
+import React, { useState, useContext } from 'react'
 import {useA11yRouteChange, useDocumentTitle} from 'components'
 import {
   PageSection,
   TextContent,
   Title,
   Text,
   Card,
-  CardBody
+  CardBody,
+  Modal,
+  Button
 } from '@patternfly/react-core'
 
+import moment from 'moment'
+import { I18nContext } from 'i18n/I18nProvider'
+import { 
+  OVERVIEW as STRINGS_ENG,
+  COMMONS as COMMONS_ENG
+} from 'i18n/locales/en/strings'
+import { 
+  OVERVIEW as STRINGS_JAP,
+  COMMONS as COMMONS_JAP
+} from 'i18n/locales/jp/strings'
+
 const Overview: React.FunctionComponent = ({children}) => {
+  const [strings, setStrings] = useState(STRINGS_ENG)
+  const [commonStrings, setCommonStrings] = useState(COMMONS_ENG)
+  const [locale, setLocale] = useState('en')
+  const polyglot = useContext(I18nContext)
+  polyglot.locale(locale)  
+  polyglot.extend({ ...strings, ...commonStrings })
+
+  const onClick = (event: React.SyntheticEvent<HTMLButtonElement>) => {
+    setStrings(event.currentTarget.value === 'en' ? STRINGS_ENG : STRINGS_JAP)
+    setCommonStrings(event.currentTarget.value === 'en' ? COMMONS_ENG : COMMONS_JAP)
+    setLocale(event.currentTarget.value)
+  }
+
   useA11yRouteChange()
-  useDocumentTitle('Overview')
+  useDocumentTitle(polyglot.t('page_title'))
   return (
     <>
       <PageSection variant={'light'}>
         <TextContent>
-          <Title size={'3xl'}>Overview</Title>
+          <p>
+            <button value='en' onClick={onClick}>English</button>
+            <button value='jp' onClick={onClick}>日本語</button>
+          </p>
+          <Title size={'3xl'}>{polyglot.t('body_title')}</Title>
           <Text>
-            <b>PortaFly</b> the next gen UI for Porta API MGMT App
+          <b>{polyglot.t('subtitle')}</b>     
```

Comment:
How do we cope with partially-bold phrases like this one?

Reply:
Polyglot doesn't provide any HTML format support (AFAIK). So, we need to break it into several fields:
`<b>{polyglot.t('subtitle.bold_word')}</b>{polyglot.t('subtitle.normal_word')}`

Reply:
That's inconvenient :/